dashboard/serializers_test.py

```diff
@@ -146,20 +163,38 @@ def test_full_program_user_serialization_financial_aid(self):
         the difference with test_full_program_user_serialization
         is that the grade is calculated using the current grades
         """
+        Profile.objects.filter(pk=self.profile.pk).update(email_optin=False)
+        self.profile.refresh_from_db()
         expected_result = {
             'id': self.fa_program.id,
             'enrollments': list(CachedEnrollment.active_data(self.user, self.fa_program)),
             'certificates': [],
             'current_grades': list(CachedCurrentGrade.active_data(self.user, self.fa_program)),
             'grade_average': 95,
-            'is_learner': True
+            'is_learner': True,
+            'email_optin': False
+        }
+        assert UserProgramSearchSerializer.serialize(self.fa_program_enrollment) == expected_result
+
+        Profile.objects.filter(pk=self.profile.pk).update(email_optin=True)
```

Comment:
Actually I don't think we need to test `email_optin` here or below. We only need to test it in one place, it's not linked to any other field in the profile.